### PR TITLE
update CvssV3VulnAssessmentRelationship.md

### PR DESCRIPTION
**Pull request by jeff-schutt**
These updates address changes to CVSSv3VulnAssessmentRelationship.md in support of https://github.com/spdx/spdx-3-model/pull/501#pullrequestreview-1651152833

CC @goneall, @rnjudge, @tsteenbe

### CODE REVIEW

**Review by goneall — APPROVED** [review]
LGTM

File: model/Security/Classes/CvssV3VulnAssessmentRelationship.md [hunk 1]
```diff
@@ -4,20 +4,18 @@ SPDX-License-Identifier: Community-Spec-1.0
...
+Provides a CVSS version 3 assessment for a vulnerability.
 
 ## Description
 
 A CvssV3VulnAssessmentRelationship relationship describes the determined score,
-severity, and vector of a vulnerability using version 3.1 of the Common
-Vulnerability Scoring System (CVSS) as defined on 
-[https://www.first.org/cvss/v3.1/specification-document](https://www.first.org/cvss/v3.1/specification-document). It is intented to communicate the results of using a CVSS calculator.
+severity, and vector of a vulnerability using version [3.0](https://www.first.org/cvss/v3.0/specification-document) or [3.1](https://www.first.org/cvss/v3.1/specification-document) of the Common
+Vulnerability Scoring System (CVSS). It is intented to communicate the results of using a CVSS calculator.
```
Comment (jeff-schutt) [inline model/Security/Classes/CvssV3VulnAssessmentRelationship.md:13]:
```suggestion
Vulnerability Scoring System (CVSS). It is intended to communicate the results of using a CVSS calculator.
```

**Review by kestewart — APPROVED** [review]
We're missing how to enforce to/from constraints, but not blocking on this.

**Comment by jeff-schutt** [comment]
Discussed in the security meeting and agreed to add these constraints to the abstract class `VulnAssessmentRelationship`

**Pull request merged**